resources/pithos.yaml

```diff
@@ -99,7 +124,7 @@ spec:
       product: pithos
       pithos-role: pithos
       component: pithos
-  replicas: 1
+  replicas: 2
```

Comment:
Just curious, why this change?

Reply:
Typically 1 is not enough and we don't have anyone one node profile for the app anyway.